src/workflow/confirm.c

```diff
@@ -105,11 +105,16 @@ enum workflow_async_ready workflow_confirm_async(
     }
 }
 
-bool workflow_confirm(const char* title, const char* body, bool longtouch, bool accept_only)
+bool workflow_confirm(
+    const char* title,
+    const char* body,
+    bool longtouch,
+    bool monospaced,
```

Comment:
What about providing directly the desired font, like here? https://github.com/digitalbitbox/bitbox02-firmware/blob/4dfc14acc8d38f61e6c5e66c495b0325eddbe9df/src/ui/components/label.c#L286

Can be passed down all the way to the confirm label component, which default to the old one if NULL.

Reply:
ack